Bio/AlignIO/MafIO.py

```diff
@@ -0,0 +1,581 @@
+# Copyright 2011, 2012 by Andrew Sczesnak.  All rights reserved.
+# Revisions Copyright 2011, 2017 by Peter Cock.  All rights reserved.
+# Revisions Copyright 2014, 2015 by Adam Novak.  All rights reserved.
+#
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+"""Bio.AlignIO support for the "maf" multiple alignment format.
+
+You are expected to use this module via the Bio.AlignIO functions(or the
+Bio.SeqIO functions if you want to work directly with the gapped sequences).
+
+"""
+from Bio.Alphabet import single_letter_alphabet
+from Bio.Seq import Seq
+from Bio.SeqRecord import SeqRecord
+from Bio.Align import MultipleSeqAlignment
+from .Interfaces import SequentialAlignmentWriter
+import shlex
```

Comment:
Good catch. I've recently started using flake8 on the codebase and found (and fixed) other unused imports in the code base.